src/main/java/com/hedvig/notificationService/customerio/events/ActivationDateTodayEvent.kt

```diff
@@ -0,0 +1,12 @@
+package com.hedvig.notificationService.customerio.events
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.fasterxml.jackson.databind.PropertyNamingStrategy
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy::class)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "name", visible = true)
+data class ActivationDateTodayEvent(
```

Comment:
Is the naming a bit confusing? Shall we following the naming that we are using in the events for far. 
```suggestion
data class ContractsActivatedTodayEvent(
```

Reply:
Good idea.